staging/src/k8s.io/component-base/version/verflag/verflag.go

```diff
@@ -25,19 +25,17 @@ import (
 
 	flag "github.com/spf13/pflag"
 
-	"k8s.io/component-base/version"
+	k8sversion "k8s.io/component-base/version"
 )
 
-type versionValue int
+type versionValue string
 
 const (
-	VersionFalse versionValue = 0
-	VersionTrue  versionValue = 1
-	VersionRaw   versionValue = 2
+	versionFalse versionValue = "false"
+	versionTrue  versionValue = "true"
+	versionRaw   versionValue = "raw"
 )
 
```

Comment:
seems that ‘int’ to 'string' change do in others PR better? this PR just  change Public type to Private type